components/ExplorerChartSection.js

```diff
@@ -92,7 +95,15 @@ const ChartSection = observer(
             )}
           </Box>
         </Box>
-        <Typography variant="body1" color="textSecondary" component="span">
+        {dimension == "chart" && (
+          <ExplorerLegend data={data} activeGroup={explorerStore.activeGroup} />
+        )}
+        <Typography
+          variant="body1"
+          color="textSecondary"
+          component="span"
+          style={{ textAlign: "right" }}
```

Comment:
```suggestion
          align="right"
```

See: https://material-ui.com/api/typography/